packages/manager/modules/nutanix/src/dashboard/general-info/constants.js

```diff
@@ -3,44 +3,29 @@ export const TRAVAUX_LINK = {
-  DE: 'https://www.ovhcloud.com/de/hosted-private-cloud/nutanix/',
-  ES: 'https://www.ovhcloud.com/es-es/hosted-private-cloud/nutanix/',
-  FR: 'https://www.ovhcloud.com/fr/hosted-private-cloud/nutanix/',
-  GB: 'https://www.ovhcloud.com/en-gb/hosted-private-cloud/nutanix/',
-  IE: 'https://www.ovhcloud.com/en-ie/hosted-private-cloud/nutanix/',
-  IT: 'https://www.ovhcloud.com/it/hosted-private-cloud/nutanix/',
-  NL: 'https://www.ovhcloud.com/nl/hosted-private-cloud/nutanix/',
-  PL: 'https://www.ovhcloud.com/pl/hosted-private-cloud/nutanix/',
-  PT: 'https://www.ovhcloud.com/pt/hosted-private-cloud/nutanix/',
-  MA: 'https://www.ovhcloud.com/fr-ma/hosted-private-cloud/nutanix/',
-  SN: 'https://www.ovhcloud.com/fr-sn/hosted-private-cloud/nutanix/',
-  TN: 'https://www.ovhcloud.com/fr-tn/hosted-private-cloud/nutanix/',
-  WS: 'https://www.ovhcloud.com/es/hosted-private-cloud/nutanix/',
-  US: 'https://us.ovhcloud.com/hosted-private-cloud/nutanix/',
-  DEFAULT: 'https://www.ovhcloud.com/en-gb/hosted-private-cloud/nutanix/',
+export const GUIDE_PACKAGES_BASE_URL = {
+  CA: 'https://www.ovhcloud.com/en-ca/hosted-private-cloud/nutanix',
+  QC: 'https://www.ovhcloud.com/fr-ca/hosted-private-cloud/nutanix',
+  DE: 'https://www.ovhcloud.com/de/hosted-private-cloud/nutanix',
+  ES: 'https://www.ovhcloud.com/es-es/hosted-private-cloud/nutanix',
+  FR: 'https://www.ovhcloud.com/fr/hosted-private-cloud/nutanix',
+  GB: 'https://www.ovhcloud.com/en-gb/hosted-private-cloud/nutanix',
+  IE: 'https://www.ovhcloud.com/en-ie/hosted-private-cloud/nutanix',
+  IT: 'https://www.ovhcloud.com/it/hosted-private-cloud/nutanix',
+  NL: 'https://www.ovhcloud.com/nl/hosted-private-cloud/nutanix',
+  PL: 'https://www.ovhcloud.com/pl/hosted-private-cloud/nutanix',
+  PT: 'https://www.ovhcloud.com/pt/hosted-private-cloud/nutanix',
+  MA: 'https://www.ovhcloud.com/fr-ma/hosted-private-cloud/nutanix',
+  SN: 'https://www.ovhcloud.com/fr-sn/hosted-private-cloud/nutanix',
+  TN: 'https://www.ovhcloud.com/fr-tn/hosted-private-cloud/nutanix',
+  WS: 'https://www.ovhcloud.com/es/hosted-private-cloud/nutanix',
+  US: 'https://us.ovhcloud.com/hosted-private-cloud/nutanix',
+  DEFAULT: 'https://www.ovhcloud.com/en-gb/hosted-private-cloud/nutanix',
 };
 
-export const GUIDE_PACKAGES_URL = {
-  CA: `${GUIDES_URL.CA}packaged`,
-  QC: `${GUIDES_URL.QC}packaged`,
-  DE: `${GUIDES_URL.DE}packaged`,
-  ES: `${GUIDES_URL.ES}packaged`,
-  FR: `${GUIDES_URL.FR}packaged`,
-  GB: `${GUIDES_URL.GB}packaged`,
-  IE: `${GUIDES_URL.IE}packaged`,
-  IT: `${GUIDES_URL.IT}packaged`,
-  NL: `${GUIDES_URL.NL}packaged`,
-  PL: `${GUIDES_URL.PL}packaged`,
-  PT: `${GUIDES_URL.PT}packaged`,
-  MA: `${GUIDES_URL.MA}packaged`,
-  SN: `${GUIDES_URL.SN}packaged`,
-  TN: `${GUIDES_URL.TN}packaged`,
-  WS: `${GUIDES_URL.WS}packaged`,
-  US: `${GUIDES_URL.US}packaged`,
-  DEFAULT: `${GUIDES_URL.DEFAULT}packaged`,
+export const GUIDE_PACKAGES_ENDPOINT = {
+  BYOL: '/byol',
+  DEFAULT: '/packaged',
```

Comment:
packaged are for pack offers so it's not really the default one. 
```suggestion
  PACK: '/packaged',
```